de.fu_berlin.inf.dpp/test/junit/de/fu_berlin/inf/dpp/filesystem/IResourceTest.java

```diff
@@ -13,7 +13,7 @@ public void testGetAdapter() {
         org.eclipse.core.resources.IFolder folder = EasyMock
             .createMock(org.eclipse.core.resources.IFolder.class);
 
-        Capture<Class<?>> mappedAdapterClassCapture = new Capture<Class<?>>();
+        Capture<Class> mappedAdapterClassCapture = new Capture();
```

Comment:
This produces some yellow lines ... saying this statement should be generified ...

Should be Capture<Class<? extends IResource>>